theme/gatsby-node.js

```diff
@@ -68,11 +82,23 @@ function getEditUrl(repo, filePath) {
   return `https://github.com/${repo.user}/${repo.project}/edit/master/${filePath}`
 }
 
-async function fetchContributors(repo, filePath) {
+async function fetchContributors(repo, filePath, accessToken) {
+  const hash = `${repo.user}/${repo.project}/${filePath}`
+  const cached = CONTRIBUTOR_CACHE.get(hash)
+  if (cached) {
+    console.debug('found something in the cache. whee!')
+    return cached
+  }
+
   try {
-    const {data} = await axios.get(
-      `https://api.github.com/repos/${repo.user}/${repo.project}/commits?path=${filePath}`,
-    )
+    const {data} = await axios.request({
+      method: 'get',
+      baseURL: 'https://api.github.com/',
+      url: `/repos/${repo.user}/${repo.project}/commits?path=${filePath}&per_page=100`,
```

Comment:
Good call setting `per_page` to 100 and using an access token 👍 We were probably missing contributors before 😬